### PR TITLE
Sc 256 domain and entities

### PR DESCRIPTION
**Pull request by marialobillo**

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-const app: Express = express()
-
-app.get('/', (req: Request, res: Response) => {
-	res.send
-('Hiii!')
-})
-
-app.listen(port, () => {
-	console.log(`Server running on port ${port}`)
-})
```
Comment (AbdelrahmanRadwan) [inline server/index.ts:13]: Why is this deleted btw?

File: server/build/index.js [hunk 2]
```diff
@@ -1,14 +0,0 @@
...
-Object.defineProperty(exports, "__esModule", { value: true });
-const express_1 = __importDefault(require("express"));
-const port = 8000;
-const app = (0, express_1.default)();
-app.get('/', (req, res) => {
-    res.send('Hiii!');
-});
-app.listen(port, () => {
-    console.log(`Server running on port ${port}`);
-});
```
Comment (AbdelrahmanRadwan) [inline server/build/index.js:14]: why do we delete?

File: server/index.ts (hunk 1 above)
Comment (marialobillo) [reply server/index.ts:13]: I just deleted because I re-started from another approach, I guess we can talk about this on Monday
